.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: Lint GitHub Actions workflows
+on:
+  push:
+    branches:
+      - "main"
+      - "release-**"
+    paths:
+      - 'actions/**/*.ya?ml' # Any in-house action
+      - '.github/workflows/*.ya?ml'
+      - '.github/workflows/actionlint.*' # This workflow
+  pull_request:
+    branches:
+      - "main"
+      - "release-**"
+    paths:
+      - 'actions/**/*.ya?ml' # Any in-house action
+      - '.github/workflows/*.ya?ml'
+      - '.github/workflows/actionlint.*' # This workflow
+
+env:
+  LC_ALL: en_US.UTF-8
```

Comment:
Do we need this?

Reply:
good point! i missed that when copying from the other repo. fixed!